eli5/formatters/html.py

```diff
@@ -27,13 +28,15 @@
     remaining_weight_color=lambda ws, w_range, pos_neg:
         format_hsl(remaining_weight_color_hsl(ws, w_range, pos_neg)),
     format_feature=lambda f, w, hl: _format_feature(f, w, hl_spaces=hl),
+    format_value=format_value,
     format_decision_tree=lambda tree: _format_decision_tree(tree),
 ))
 
 
 def format_as_html(explanation, include_styles=True, force_weights=True,
                    show=fields.ALL, preserve_density=None,
-                   highlight_spaces=None, horizontal_layout=True):
+                   highlight_spaces=None, horizontal_layout=True,
+                   show_feature_values=True):
```

Comment:
That's true for linear models, but for tree ensembles it's not: they don't have weights, only feature importances. And even if we somehow could calculate the weights, it won't be true I think.
But we can add "(for linear models, contribution = weight * feature value)", I think it will clarify what is contribution even for non-linear ones.